src/auro-checkbox.js

```diff
@@ -50,6 +50,15 @@ class AuroCheckbox extends LitElement {
     };
   }
 
+  // This custom event is only for the purpose of supporting IE
+  // .addEventListener('change', function() { })
+  change(event) {
+    this.value = event.target.value
+    const customEvent = new event.constructor(event.type, event)
```

Comment:
```suggestion
    const customEvent = new CustomEvent(event.type, event);
```
For some reason this line was throwing an error in IE, despite everything being polyfilled. Changing to use the CustomEvent constructor made it work.